includes/templates/reader-activation-emails/password-reset.php

```diff
@@ -60,7 +60,7 @@
 	$social_links .
 
 	'<!-- wp:paragraph -->
```

Comment:
Not sure why this is, but this missing information in the `wp:paragraph` comment is making it so this footer isn't as small as the others for me when the email is sent (looks fine in the editor though!).

```suggestion
	'<!-- wp:paragraph {"fontSize":"small"} -->
```

![CleanShot 2024-08-14 at 17 15 56](https://github.com/user-attachments/assets/1e18457a-028a-49f8-9095-fa6acc3115b2)

Reply:
Good catch! fixed in https://github.com/Automattic/newspack-plugin/pull/3335/commits/16f31d0ae4a46f92b36c9d338773d751a0aea180